source/unified-test-format/unified-test-format.rst

```diff
@@ -1779,6 +1779,54 @@ The following arguments are supported:
 
 - ``session``: Optional string. See `commonOptions_session`_.
 
+runCursorCommand
+~~~~~~~~~~~~~~~~
+
+`Generic cursor returning command runner <../run-command/run-command.rst>.`__
+
+This method does not inherit a read concern or write concern (per the
+`Read and Write Concern <../read-write-concern/read-write-concern.rst#generic-command-method>`__
+spec), nor does it inherit a read preference (per the
+`Server Selection <../server-selection/server-selection.rst#use-of-read-preferences-with-commands>`__
+spec); however, they may be specified as arguments.
+
+This operation proxies the databases's ``runCursorCommand`` method and supports the same arguments and options.
+
+When executing the provided command, the test runner MUST fully iterate the cursor.
+This will ensure consistent behavior between drivers that eagerly create a server-side cursor and those that do so lazily when iteration begins.
+
+The following arguments are supported:
+
+- ``command``: Required document. The command to be executed.
+
+- ``commandName``: Required string. The name of the command to run. This is used
+  by languages that are unable preserve the order of keys in the ``command``
+  argument when parsing YAML/JSON.
+
+- ``readPreference``: Optional object. See `commonOptions_readPreference`_.
+
+- ``session``: Optional string. See `commonOptions_session`_.
+
+- ``batchSize``: Optional int32. Use this value to configure the batchSize sent on the getMore command.
+
+- ``maxTimeMS``: Optional int32. Use this value to configure the maxTimeMS sent on the getMore command.
+
+- ``comment``: Optional BSON value. Use this value to configure the comment sent on the getMore command.
+
+- ``cursorType``: Optional string enum value, one of ``'tailable' | 'tailableAwait' | 'nonTailable'``. Use this value to configure the enum passed to the ``cursorType`` option.
+
+- ``timeoutMode``: Optional string enum value, one of ``'iteration' | 'cursorLifetime'``. Use this value to configure the enum passed to the ``timeoutMode`` option.
+
+createRunCursorCommand
```

Comment:
I suggest calling this `createCommandCursor` for consistency with `createFindCursor`.

This also warrants adding a "CommandCursor" definition to [Supported Entity Types](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#supported-entity-types). See "FindCursor" for prior art.

Once you define the CommandCursor entity, you should also add a mention of it in [Cursor Operations](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#cursor-operations) (alongside FindCursor and ChangeStream).

Reply:
Added the entity and updated cursor operations